makefile

```diff
@@ -9,6 +9,10 @@ test: all
 	$(VIRTUAL_ENV)/bin/pip install nose
 	$(VIRTUAL_ENV)/bin/nosetests -v --nologcapture
 
+lint:
```

Comment:
Ah, except this should have some dependencies: at least `requirements` and maybe another thing or two.